<launch>

    <include file="$(find PID)/launch/urdf.launch" />
    <node pkg="PID" type="servo_joint.py" name="servo_joint" />
    <node pkg="PID" type="pressure.py" name="pressure_visualizer" />

    <!-- PID node -->
    <node pkg="PID" type="pid.py" name="PID" output="screen" />

    <!-- <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot" args="topics /load /servo" /> -->

</launch>